pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -1583,31 +1591,40 @@ async def run(self):
                     content_q_by_type[model_type] = content_q_by_type[model_type] | unit_q
                     d_content_by_nat_key[d_content.content.natural_key()].append(d_content)
 
-            for model_type, content_q in content_q_by_type.items():
```

Comment:
Why do away with this in favor of the more verbose manual approach?

Reply:
At one point, it was because order-by-type looked like it was going to fix the problem. Now? It's because "adjusting this isn't going to 'fix' anything, and will require retesting the whole patch on an actual satellite-install to make sure it hasn't caused a problem".